test/integration/StreamrClient.test.js

```diff
@@ -16,7 +16,7 @@ describe('StreamrClient', () => {
         url: config.websocketUrl,
         restUrl: config.restUrl,
         auth: {
-            privateKey: '12345564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
+            privateKey: '0x12345564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
```

Comment:
The private key is often presented also without the `0x` in the beginning. Can we make it work both ways?